mongorestore/mongorestore_test.go

```diff
@@ -723,3 +979,48 @@ func TestAutoIndexIdNonLocalDB(t *testing.T) {
 		}
 	})
 }
+
+// TestSkipSystemCollections asserts that certain system collections like "config.systems.sessions" and the transaction
+// related tables aren't applied via applyops when replaying the oplog.
+func TestSkipSystemCollections(t *testing.T) {
```

Comment:
Per offline discussion, let's rework this test to actually query the oplog instead of just comparing collections.